tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js

```diff
@@ -61,23 +52,10 @@ const locationParams = {
 const testsToRun = [{
     from: 'LocalStorage',
     to: 'AWS',
-}, {
```

Comment:
should we add a todo here to keep track of what needs to be supported?

Reply:
Yes, created a ticket and added TODO with the ticket id.